### DISCUSSION TITLE
Command line arguments

### DISCUSSION THREAD

**Discussion by Enchillada411** [genius257/vscode-autoit#49] [Q&A]
How is it possible to use command line arguments when using the plugin in VS Code?

**Comment by genius257** [comment]
I'm not sure in what context?
If you mean running scripts, that's the https://github.com/genius257/vscode-autoit3-debug extension, but the feature for arguments via the launch configuration file is currently missing. Either way I've created the issue for that here: https://github.com/genius257/vscode-autoit3-debug/issues/12

**Reply by Enchillada411** [reply]
Yes, using properties in configuration file is exactly what is was expecting to work for the launchers when I used the the plugin (vscode-autoit3-debug). Thanks for your answer. I am looking forward for this feature.

**Reply by genius257** [reply]
`v1.1.0` of `vscode-autoit3-debug` is now released 😄 
Arguments should now be supported in the launch configuration file, via the `"arguments"` property:
```
"arguments": ["test"]
```
It expects an array of strings, please let me know if you experience any problems! 😃
